### ISSUE TITLE
Crash When Player Death Occurs

### ISSUE THREAD

**Issue by Underplayer97** [Asek3/Oculus#389]
### Minecraft Version

1.20.1

### Oculus Version

oculus-mc1.20-1.6.4

### Sodium Version

rubidium-0.6.5.jar

### Operating System

Windows 11

### What is your GPU?

NVIDA GeForce RTX 3060

### Java Version

Java 17

### Reproduction Steps

1. Fall or cause any Player Death
3. Game Crashes

### Crash Report file and latest.log

```shell
---- Minecraft Crash Report ----
// You should try our sister game, Minceraft!

Time: 2023-06-24 06:06:31
Description: Unexpected error

java.lang.NullPointerException: Cannot invoke "net.minecraft.client.renderer.RenderType.m_110185_()" because "type" is null
	at net.coderbot.batchedentityrendering.impl.FullyBufferedMultiBufferSource.m_109911_(FullyBufferedMultiBufferSource.java:136) ~[oculus-mc1.20-1.6.4.jar%23173!/:?] {re:mixin,re:classloading}
	at net.minecraft.client.renderer.LevelRenderer.m_109599_(LevelRenderer.java:1377) ~[client-1.20.1-20230612.114412-srg.jar%23178!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mixins.oculus.vertexformat.json:immediate.MixinLevelRenderer,pl:mixin:APP:oculus-batched-entity-rendering.mixins.json:MixinLevelRenderer,pl:mixin:APP:oculus-batched-entity-rendering.mixins.json:MixinLevelRenderer_EntityListSorting,pl:mixin:APP:rubidium.mixins.json:features.buffer_builder.intrinsics.MixinWorldRenderer,pl:mixin:APP:rubidium.mixins.json:features.chunk_rendering.MixinWorldRenderer,pl:mixin:APP:rubidium.mixins.json:features.clouds.MixinWorldRenderer,pl:mixin:APP:rubidium.mixins.json:features.options.MixinWorldRenderer,pl:mixin:APP:mixins.oculus.json:LevelRendererAccessor,pl:mixin:APP:mixins.oculus.json:MixinLevelRenderer,pl:mixin:APP:mixins.oculus.json:fabulous.MixinDisableFabulousGraphics,pl:mixin:APP:mixins.oculus.json:shadows.MixinLevelRenderer,pl:mixin:APP:mixins.oculus.json:sky.MixinLevelRenderer_SunMoonToggle,pl:mixin:APP:mixins.oculus.fantastic.json:MixinLevelRenderer,pl:mixin:APP:mixins.oculus.compat.sodium.json:sky.MixinLevelRenderer,pl:mixin:APP:mixins.oculus.json:shadows.MixinPreventRebuildNearInShadowPass,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.GameRenderer.m_109089_(GameRenderer.java:1126) ~[client-1.20.1-20230612.114412-srg.jar%23178!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:okzoomer.mixins.json:GameRendererMixin,pl:mixin:APP:mixins.oculus.json:GameRendererAccessor,pl:mixin:APP:mixins.oculus.json:MixinGameRenderer,pl:mixin:APP:mixins.oculus.json:MixinModelViewBobbing,pl:mixin:APP:mixins.oculus.json:MixinTweakFarPlane,pl:mixin:APP:mixins.oculus.json:MixinGameRenderer_NightVisionCompat,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.GameRenderer.m_109093_(GameRenderer.java:909) ~[client-1.20.1-20230612.114412-srg.jar%23178!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:okzoomer.mixins.json:GameRendererMixin,pl:mixin:APP:mixins.oculus.json:GameRendererAccessor,pl:mixin:APP:mixins.oculus.json:MixinGameRenderer,pl:mixin:APP:mixins.oculus.json:MixinModelViewBobbing,pl:mixin:APP:mixins.oculus.json:MixinTweakFarPlane,pl:mixin:APP:mixins.oculus.json:MixinGameRenderer_NightVisionCompat,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1146) ~[client-1.20.1-20230612.114412-srg.jar%23178!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:rubidium.mixins.json:core.MixinMinecraftClient,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Keybinds,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Images,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_PipelineManagement,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91374_(Minecraft.java:718) ~[client-1.20.1-20230612.114412-srg.jar%23178!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:rubidium.mixins.json:core.MixinMinecraftClient,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Keybinds,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Images,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_PipelineManagement,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:218) ~[forge-47.0.19.jar:?] {re:classloading,pl:runtimedistcleaner:A}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.runTarget(CommonLaunchHandler.java:111) ~[fmlloader-1.20.1-47.0.19.jar:?] {}
	at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.clientService(CommonLaunchHandler.java:99) ~[fmlloader-1.20.1-47.0.19.jar:?] {}
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$makeService$0(CommonClientLaunchHandler.java:25) ~[fmlloader-1.20.1-47.0.19.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:108) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:78) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141) ~[bootstraplauncher-1.1.2.jar:?] {}


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.coderbot.batchedentityrendering.impl.FullyBufferedMultiBufferSource.m_109911_(FullyBufferedMultiBufferSource.java:136) ~[oculus-mc1.20-1.6.4.jar%23173!/:?] {re:mixin,re:classloading}
	at net.minecraft.client.renderer.LevelRenderer.m_109599_(LevelRenderer.java:1377) ~[client-1.20.1-20230612.114412-srg.jar%23178!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mixins.oculus.vertexformat.json:immediate.MixinLevelRenderer,pl:mixin:APP:oculus-batched-entity-rendering.mixins.json:MixinLevelRenderer,pl:mixin:APP:oculus-batched-entity-rendering.mixins.json:MixinLevelRenderer_EntityListSorting,pl:mixin:APP:rubidium.mixins.json:features.buffer_builder.intrinsics.MixinWorldRenderer,pl:mixin:APP:rubidium.mixins.json:features.chunk_rendering.MixinWorldRenderer,pl:mixin:APP:rubidium.mixins.json:features.clouds.MixinWorldRenderer,pl:mixin:APP:rubidium.mixins.json:features.options.MixinWorldRenderer,pl:mixin:APP:mixins.oculus.json:LevelRendererAccessor,pl:mixin:APP:mixins.oculus.json:MixinLevelRenderer,pl:mixin:APP:mixins.oculus.json:fabulous.MixinDisableFabulousGraphics,pl:mixin:APP:mixins.oculus.json:shadows.MixinLevelRenderer,pl:mixin:APP:mixins.oculus.json:sky.MixinLevelRenderer_SunMoonToggle,pl:mixin:APP:mixins.oculus.fantastic.json:MixinLevelRenderer,pl:mixin:APP:mixins.oculus.compat.sodium.json:sky.MixinLevelRenderer,pl:mixin:APP:mixins.oculus.json:shadows.MixinPreventRebuildNearInShadowPass,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.GameRenderer.m_109089_(GameRenderer.java:1126) ~[client-1.20.1-20230612.114412-srg.jar%23178!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:okzoomer.mixins.json:GameRendererMixin,pl:mixin:APP:mixins.oculus.json:GameRendererAccessor,pl:mixin:APP:mixins.oculus.json:MixinGameRenderer,pl:mixin:APP:mixins.oculus.json:MixinModelViewBobbing,pl:mixin:APP:mixins.oculus.json:MixinTweakFarPlane,pl:mixin:APP:mixins.oculus.json:MixinGameRenderer_NightVisionCompat,pl:mixin:A,pl:runtimedistcleaner:A}
-- Affected level --
Details:
	All players: 1 total; [LocalPlayer['Underplayer97'/1208089, l='ClientLevel', x=7823.28, y=162.00, z=-4848.49]]
	Chunk stats: 1024, 453
	Level dimension: minecraft:overworld
	Level spawn location: World: (112,70,367), Section: (at 0,6,15 in 7,4,22; chunk contains blocks 112,-64,352 to 127,319,367), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,-64,0 to 511,319,511)
	Level time: 313654585 game time, 330331473 day time
	Server brand: Paper
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.client.multiplayer.ClientLevel.m_6026_(ClientLevel.java:455) ~[client-1.20.1-20230612.114412-srg.jar%23178!/:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:mixin:APP:rubidium.mixins.json:features.chunk_rendering.MixinClientWorld,pl:mixin:APP:rubidium.mixins.json:features.fast_biome_colors.MixinClientWorld,pl:mixin:APP:mixins.oculus.vertexformat.json:block_rendering.MixinClientLevel,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91354_(Minecraft.java:2319) ~[client-1.20.1-20230612.114412-srg.jar%23178!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:rubidium.mixins.json:core.MixinMinecraftClient,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Keybinds,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Images,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_PipelineManagement,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91374_(Minecraft.java:740) ~[client-1.20.1-20230612.114412-srg.jar%23178!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:rubidium.mixins.json:core.MixinMinecraftClient,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Keybinds,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Images,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_PipelineManagement,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:218) ~[forge-47.0.19.jar:?] {re:classloading,pl:runtimedistcleaner:A}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.runTarget(CommonLaunchHandler.java:111) ~[fmlloader-1.20.1-47.0.19.jar:?] {}
	at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.clientService(CommonLaunchHandler.java:99) ~[fmlloader-1.20.1-47.0.19.jar:?] {}
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$makeService$0(CommonClientLaunchHandler.java:25) ~[fmlloader-1.20.1-47.0.19.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:108) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:78) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141) ~[bootstraplauncher-1.1.2.jar:?] {}


-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: vanilla, mod_resources

-- System Details --
Details:
	Minecraft Version: 1.20.1
	Minecraft Version ID: 1.20.1
	Operating System: Windows 11 (amd64) version 10.0
	Java Version: 17.0.3, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 457497056 bytes (436 MiB) / 1040187392 bytes (992 MiB) up to 8522825728 bytes (8128 MiB)
	CPUs: 8
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 3 3100 4-Core Processor              
	Identifier: AuthenticAMD Family 23 Model 113 Stepping 0
	Microarchitecture: Zen 2
	Frequency (GHz): 3.59
	Number of physical packages: 1
	Number of physical CPUs: 4
	Number of logical CPUs: 8
	Graphics card #0 name: NVIDIA GeForce RTX 3060 Ti
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 4095.00
	Graphics card #0 deviceId: 0x2414
	Graphics card #0 versionInfo: DriverVersion=31.0.15.3179
	Memory slot #0 capacity (MB): 8192.00
	Memory slot #0 clockSpeed (GHz): 2.13
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 8192.00
	Memory slot #1 clockSpeed (GHz): 2.13
	Memory slot #1 type: DDR4
	Memory slot #2 capacity (MB): 8192.00
	Memory slot #2 clockSpeed (GHz): 2.13
	Memory slot #2 type: DDR4
	Memory slot #3 capacity (MB): 8192.00
	Memory slot #3 clockSpeed (GHz): 2.13
	Memory slot #3 type: DDR4
	Virtual memory max (MB): 65482.94
	Virtual memory used (MB): 44232.24
	Swap memory total (MB): 32768.00
	Swap memory used (MB): 1351.09
	JVM Flags: 4 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx8128m -Xms256m
	Loaded Shaderpack: (off)
	NEC status: No NEC detected
	Launched Version: forge-47.0.19
	Backend library: LWJGL version 3.3.1 build 7
	Backend API: NVIDIA GeForce RTX 3060 Ti/PCIe/SSE2 GL version 4.6.0 NVIDIA 531.79, NVIDIA Corporation
	Window size: 1920x1009
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'forge'
	Type: Client (map_client.txt)
	Graphics mode: fancy
	Resource Packs: 
	Current Language: en_us
	CPU: 8x AMD Ryzen 3 3100 4-Core Processor 
	ModLauncher: 10.0.9+10.0.9+main.dcd20f30
	ModLauncher launch target: forgeclient
	ModLauncher naming: srg
	ModLauncher services: 
		mixin-0.8.5.jar mixin PLUGINSERVICE 
		eventbus-6.0.3.jar eventbus PLUGINSERVICE 
		fmlloader-1.20.1-47.0.19.jar slf4jfixer PLUGINSERVICE 
		fmlloader-1.20.1-47.0.19.jar object_holder_definalize PLUGINSERVICE 
		fmlloader-1.20.1-47.0.19.jar runtime_enum_extender PLUGINSERVICE 
		fmlloader-1.20.1-47.0.19.jar capability_token_subclass PLUGINSERVICE 
		accesstransformers-8.0.4.jar accesstransformer PLUGINSERVICE 
		fmlloader-1.20.1-47.0.19.jar runtimedistcleaner PLUGINSERVICE 
		modlauncher-10.0.9.jar mixin TRANSFORMATIONSERVICE 
		modlauncher-10.0.9.jar fml TRANSFORMATIONSERVICE 
	FML Language Providers: 
		minecraft@1.0
		lowcodefml@null
		javafml@null
	Mod List: 
		client-1.20.1-20230612.114412-srg.jar             |Minecraft                     |minecraft                     |1.20.1              |DONE      |Manifest: a1:d4:5e:04:4f:d3:d6:e0:7b:37:97:cf:77:b0:de:ad:4a:47:ce:8c:96:49:5f:0a:cf:8c:ae:b2:6d:4b:8a:3f
		okzoomer-forge-1.20-3.0.0.jar                     |OkZoomer                      |okzoomer                      |3.0.0               |DONE      |Manifest: NOSIGNATURE
		voicechat-forge-1.20.1-2.4.11.jar                 |Simple Voice Chat             |voicechat                     |1.20.1-2.4.11       |DONE      |Manifest: NOSIGNATURE
		MouseTweaks-forge-mc1.20-2.25.jar                 |Mouse Tweaks                  |mousetweaks                   |2.25                |DONE      |Manifest: NOSIGNATURE
		betterfpsdist-1.20.1-2.9.jar                      |betterfpsdist mod             |betterfpsdist                 |1.20.1-2.9          |DONE      |Manifest: NOSIGNATURE
		forge-1.20.1-47.0.19-universal.jar                |Forge                         |forge                         |47.0.19             |DONE      |Manifest: 84:ce:76:e8:45:35:e4:0e:63:86:df:47:59:80:0f:67:6c:c1:5f:6e:5f:4d:b3:54:47:1a:9f:7f:ed:5e:f2:90
		rubidium-0.6.5.jar                                |Rubidium                      |rubidium                      |0.6.5               |DONE      |Manifest: NOSIGNATURE
		oculus-mc1.20-1.6.4.jar                           |Oculus                        |oculus                        |1.6.4               |DONE      |Manifest: NOSIGNATURE
		ferritecore-6.0.0-forge.jar                       |Ferrite Core                  |ferritecore                   |6.0.0               |DONE      |Manifest: 41:ce:50:66:d1:a0:05:ce:a1:0e:02:85:9b:46:64:e0:bf:2e:cf:60:30:9a:fe:0c:27:e0:63:66:9a:84:ce:8a
		journeymap-1.20.1-5.9.9-forge.jar                 |Journeymap                    |journeymap                    |5.9.9               |DONE      |Manifest: NOSIGNATURE
		Controlling-forge-1.20.1-12.0.1.jar               |Controlling                   |controlling                   |12.0.1              |DONE      |Manifest: NOSIGNATURE
		Searchables-forge-1.20.1-1.0.1.jar                |Searchables                   |searchables                   |1.0.1               |DONE      |Manifest: NOSIGNATURE
		jei-1.20.1-forge-15.1.0.19.jar                    |Just Enough Items             |jei                           |15.1.0.19           |DONE      |Manifest: NOSIGNATURE
	Crash Report UUID: a51d24a1-04de-4993-8238-d3cabd8b776c
	FML: 47.0
	Forge: net.minecraftforge:47.0.19
```

### Additional context

When rejoining, causes player to crash again

**Comment by PeppermintFelix** [comment]
Hello, I am also experiencing this issue but after some experimenting with the mods we have in common I've discovered removing journey map completely fixes this issue, so it seems to be a compatibility issue with journey map.

**Comment by Underplayer97** [comment]
> Hello, I am also experiencing this issue but after some experimenting with the mods we have in common I've discovered removing journey map completely fixes this issue, so it seems to be a compatibility issue with journey map.

After testing more with Journey Map, it seems to crash when the game renders a waypoint. When a player dies and hits respawn, Journey Map makes a new waypoint highlighting the location of your death which causes the rendering bug and crashes the client

**Comment by Underplayer97** [comment]
A temporary fix for this is to not use the waypoint system on Journey Map or use Xaero's Minimap instead

**Comment by mysticdrew** [comment]
This is an issue with JourneyMap, it is fixed in `JourneyMap-1.20.1-5.9.10-forge` should be on curseforge once approved(couple hours from now) and modrinth very soon (couple minutes)
